server/plugin.go

```diff
@@ -81,13 +85,31 @@ func (p *Plugin) OnActivate() error {
 		return err
 	}
 	p.API.RegisterCommand(getCommand())
-	user, err := p.API.GetUserByUsername(config.Username)
+
+	botId, appErr := p.Helpers.EnsureBot(&model.Bot{
+		Username:    "github",
```

Comment:
There's a non-zero chance someone used to customized this away from `github`. Worth updating the README.md to help guide those upgrading? Ideally they would rename the existing user to github to preserve the "flow" from past message interactions.

Reply:
Ya. I need to upgrade the README to make sure people do the bot account conversion if they have an existing account named `github`. Will mention this.